docs/overview/app-package-for-microsoft-365.md

```diff
@@ -0,0 +1,141 @@
+
+At minimum, an app package contains:
+
+- The **app manifest** (`manifest.json`), which describes app configuration, capabilities, required resources, and important attributes
+- A **large color icon** (`color.png`), a full-color 92x92 icon to display your agent in the Microsoft 365 Copilot UI and store
+- A **small outline icon** (`outline.png`), a 32x32 icon with transparent background (not currently used in Copilot, but required to pass validation)
+
+The app package can also contain declarative agent and API plugin definitions, as well as localization files for other supported languages.
+
+:::image type="content" source="../images/app-package.png" alt-text="Diagram showing the anatomy of a Microsoft 365 app package: app manifest (.json file) + icons (color and outline .png files) wrapped in a .zip file" border="false":::
+
+## App manifest
+
+The unified app manifest for Microsoft 365 is a JSON file that describes the functionality and characteristics of your add-in, such as:
+
+- The add-in's display name, description, ID, version, and default locale.
+
+- How the add-in integrates with Office.  
+
+- How the add-in itegrates with Copilot (preview)
+
+- The permission level and data access requirements for the add-in.
+
+For a detailed overview of the manifest, see [Office Add-ins with the unified app manifest for Microsoft 365](../develop/unified-manifest-overview.md). Reference documentation for the app manifest at [The app manifest for Microsoft 365](/microsoft-365/extensibility/schema).
+
+## App icons
+
+Your app package must include both a color and outline version of your app icon, as .png files. These icons have specific size requirements in order to pass store validation.
+
+For detailed design guidance for color and outline icons for the Microsoft 365 app package, see [Design icons for add-in acquisition and management](../design/microsoft-365-extension-management-icons.md).
+
+### Color icon
+
+The color icon represents your app for Microsoft 365 within the Copilot UI and in-product (Teams, Office, Outlook, Microsoft 365) app stores.
+
+:::row:::
+:::column:::
+
+:::image type="content" source="../images/color-icon.png" alt-text="Sample image of an app color icon, showing 192x192 pixels as total icon size with background included, with a central 120x120 pixel space showing the 'Safe region' for the app symbol":::
+
+:::column-end:::
+:::column span="2":::
+
+Your color icon:
+
+- Can be any color
+- Must be 192 x 192 pixels in size
+- Should contain a symbol within 120 x 120 pixels (to allow 36 pixels of padding for [host scenarios where it is cropped](/microsoftteams/platform/concepts/build-and-test/apps-package#color-icon))
```

Comment:
Is this the correct link? <https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/apps-package#color-icon> doesn't contain a "Color icon" section.